cmd/svcat/instance/get_cmd.go

```diff
@@ -75,6 +90,24 @@ func (c *getCmd) getAll() error {
 		return err
 	}
 
+	if c.planFilter != "" {
+		instances = c.filterListByPlan(instances)
+
+		if len(instances.Items) <= 0 {
+			// All instances found were filtered out by plan
```

Comment:
I personally don't think this comment is necessary, as the code seems clear enough.